ui/app/adapters/job.js

```diff
@@ -59,14 +58,15 @@ export default Watchable.extend(WithNamespaceIDs, {
         Job: job.get('_newDefinitionJSON'),
       },
     });
-  },
+  }
 
   update(job) {
     const jobId = job.get('id') || job.get('_idBeforeSaving');
+    // debugger;
```

Comment:
I think this was from me. Whoops 😊

Reply:
Thanks for catching this, I tried to look things over before requesting review but the sheer volume is prohibitive 😳